spec/controllers/api/v1/onestop_id_controller_spec.rb

```diff
@@ -11,16 +11,16 @@
   describe 'GET show' do
     it 'will return an Operator' do
       get :show, onestop_id: @sfmta.onestop_id
-      expect_json({ onestop_id: -> (onestop_id) {
+      expect_json('operator', onestop_id: -> (onestop_id) {
         expect(onestop_id).to eq @sfmta.onestop_id
-      }})
+      })
     end
 
     it 'will return a Stop' do
       get :show, onestop_id: @metro_embarcadero.onestop_id
-      expect_json({ onestop_id: -> (onestop_id) {
+      expect_json('stop', onestop_id: -> (onestop_id) {
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Use the `lambda` method for multi-line lambdas.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.